oak_containers_hello_world_trusted_app/src/orchestrator_client.rs

```diff
@@ -70,6 +75,7 @@ impl OrchestratorClient {
     ) -> Result<CryptoContext, Box<dyn std::error::Error>> {
         let context = self
             .inner
+            // TODO(#4477): Remove unnecessary copies of the Orchestrator client.
```

Comment:
The inner tonic client is cheap to clone (it is effectively just a wrapper around an arc), so I don't think this comment is needed.